crates/bevy_sprite/src/frustum_culling.rs

```diff
@@ -0,0 +1,126 @@
+use bevy_asset::{Assets, Handle};
+use bevy_ecs::{Commands, Entity, Query, Res, With};
+use bevy_math::Vec2;
+use bevy_render::{camera::Camera, draw::Frustum};
+use bevy_transform::components::Transform;
+use bevy_window::Windows;
+
+use crate::{Sprite, TextureAtlas, TextureAtlasSprite};
+
+struct Rect {
+    position: Vec2,
+    size: Vec2,
+}
+
+impl Rect {
+    #[inline]
+    pub fn is_overlapping(&self, other: Rect) -> bool {
```

Comment:
This function could be replaced with something that allows for sprite rotation. 

1. Get each of the 4 vertices that define the corners of the sprite by applying the transform to the corners.
2. For each line of infinite length that defines the edge of the screen, check if all vertices are on the outside of the line. E.g. check if all sprite vertex x values are less than the screen's x position.
3. If they are, end early. The sprite is not in the camera frustum.
4. If the condition is not met for all four screen edges, the sprite is within the camera frustum.

Reply:
I got around this by just using a shere radius for both sprites and the camera. That means there are a few more sprites rendered than necessary, but it keeps the algorithm simple and still massively improves performance.